pkg/controller/controller_utils.go

```diff
@@ -342,6 +342,14 @@ func NewUIDTrackingControllerExpectations(ce ControllerExpectationsInterface) *U
 	return &UIDTrackingControllerExpectations{ControllerExpectationsInterface: ce, uidStore: cache.NewStore(UIDSetKeyFunc)}
 }
 
+// Reasons for pod events
+const (
+	FailedCreatePodReason     = "FailedCreate"
```

Comment:
yeah, it just feels strange to import them as `controllerutil.FailedCreatePodReason` ;-)